packages/rgbpp/tests/shared/account.ts

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
I suggest this file should be moved to `rgbpp-sdk/btc` and examples can reuse it

Reply:
Maybe we should, a concern is that we may not handle P2TR well, because it can be complicated.

Reply:
After reviewing the current codebase of the packages/examples, I think this is not a good timing to do this. Examples and tests have different needs for the feature, and it's a big change to refactor them all.